### DISCUSSION TITLE
run the pare module, I want to change the basic avatar to a woman instead of a man.

### DISCUSSION THREAD

**Discussion by pinga999** [YuliangXiu/ICON#44] [Q&A] [answered]
run the pare module, I want to change the basic avatar to a woman instead of a man.

I run demo by following repo's guide 
```
python infer.py -cfg ../configs/icon-filter.yaml -gpu 0 -in_dir ../examples -out_dir ../results -hps_type pare
```

I under stand module 'pare' use SMPL as HPS estimator
and SMPL HPS has two types for avatar and default avatar is MALE different to SMPL-X automatically detect genders

I want to know how to change option at SMPL's avatar from male to female

Thank you for referring to this discussion.

**Comment by YuliangXiu** [comment, marked as answer]
Actually, I didn't change the code of [PARE](https://github.com/mkocabas/PARE), you could ask this question in PARE's [issues](https://github.com/mkocabas/PARE/issues), or send an email directly to [Muhammed Kocabas](https://ps.is.mpg.de/person/mkocabas)
